Polyhedron/demo/Polyhedron/MainWindow.h

```diff
@@ -360,7 +360,7 @@ protected Q_SLOTS:
     QString cameraString(CGAL::Three::Viewer_interface *v) const;
   //!Hides not available operations and show available operations in all the
   //!menus.
-  void filterOperations();
+  void filterOperations(bool hide);
```

Comment:
@lrineau do you agree with this API change?

Reply:
I generally no longer like Boolean parameters. But this one is not part of any API. That is more like an ugly implementation detail. I cannot say I agree with the change, but I do not object to its merge.

Reply:
It IS an ugly workaround for a HELL of a bug, where one implementation only works on Windows and the other doesn't, and I couldn't find any reason why it doesn't work on windows, so I had to resolve to do it like that.